schema-guide.md

```diff
@@ -1224,59 +1224,199 @@ authors:
+            <li>
+              <code>value</code>:
+              <ul>
+                <li>
+                  <strong>type</strong>: <a href="#definitionsswh-identifier"><code>definitions.swh-identifier</code></a>
+                </li>
+                <li>
+                  <strong>required</strong>: <code>true</code>
+                </li>
+                <li>
+                  <strong>description</strong>: The value of the Software Heritage identifier, e.g. <code>swh:1:dir:bc286860f423ea7ced246ba7458eef4b4541cf2d</code>.
+                </li>
+              </ul>
+            </li>
+            <li>
+              <code>description</code>:
+              <ul>
+                <li>
+                  <strong>type</strong>: <a href="#definitionsidentifier-description"><code>definitions.identifier-description</code></a>
+                </li>
+                <li>
+                  <strong>required</strong>: <code>false</code>
+                </li>
+                <li>
+                  <strong>description</strong>: The description of the Software Heritage identifier, e.g. <code>The directory object of the repository as stored on Software Heritage.</code>.
+                </li>
+              </ul>
+            </li>
+          </ul>
+        </details>
+    1. <details>
+          <summary>Other</summary>
+          <br>
+          <ul>
+            <li>
+              <code>type</code>:
+              <ul>
+                <li>
+                  <strong>type</strong>: <code>enum</code> with singular value <code>other</code>
+                </li>
+                <li>
+                  <strong>required</strong>: <code>true</code>
+                </li>
+                <li>
+                  <strong>description</strong>: The type of identifier.
+                </li>
+              </ul>
+            </li>
+            <li>
+              <code>value</code>:
+              <ul>
+                <li>
+                  <strong>type</strong>: Nonempty <code>string</code>.
+                </li>
+                <li>
+                  <strong>required</strong>: <code>true</code>
+                </li>
+                <li>
+                  <strong>description</strong>: The value of the identifier, e.g. <code>arXiv:2103.06681</code>.
+                </li>
+              </ul>
+            </li>
+            <li>
+              <code>description</code>:
+              <ul>
+                <li>
+                  <strong>type</strong>: <a href="#definitionsidentifier-description"><code>definitions.identifier-description</code></a>
+                </li>
+                <li>
+                  <strong>required</strong>: <code>false</code>
+                </li>
+                <li>
+                  <strong>description</strong>: The description of the identifier, e.g. <code>The ArXiv preprint of the paper.</code>.
+                </li>
+              </ul>
+            </li>
+          </ul>
+        </details>
```

Comment:
I'm not actually sure that having this amount of HTML in the source is an improvement over the user having to scroll a bit more. This adds new 139 lines, and makes things a lot less readable.

However, I like the fact that you solved this, and I'll add comments for the case that we decide to merge this.

Reply:
It's a tradeoff for sure, but in my opinion a good one. Many readers will benefit from the easy readability in the rendered version at the cost of just a few developers (as opposed to just general readers) who now have to deal with the added madness of the collapsible nested html

Reply:
we could make it a bit denser by not breaking each list item over three lines?

Reply:
Okay, let's do it then. And yes, condensing the items a bit would be good I think. Good to go after this, thanks!

Reply:
Just pushed a denser version, revert if needed with `git revert 7cd5e86c329162e023be7792f7deb179bcbfd3a5`